de-lib/src/main/java/org/iplantc/de/apps/integration/client/view/widgets/AppTemplatePropertyEditor.java

```diff
@@ -52,30 +53,48 @@
 /**
  * @author jstroot
  */
-public class AppTemplatePropertyEditor extends Composite implements ValueAwareEditor<AppTemplate>, HasLabelOnlyEditMode, HasAppTemplateSelectedEventHandlers, ArgumentGroupSelectedEventHandler,
-                                                                    ArgumentSelectedEventHandler, HasUpdateCommandLinePreviewEventHandlers {
-
-    interface AppTemplatePropertyEditorUiBinder extends UiBinder<Widget, AppTemplatePropertyEditor> { }
+public class AppTemplatePropertyEditor extends Composite implements ValueAwareEditor<AppTemplate>,
+                                                                    HasLabelOnlyEditMode,
+                                                                    HasAppTemplateSelectedEventHandlers,
+                                                                    ArgumentGroupSelectedEventHandler,
+                                                                    ArgumentSelectedEventHandler,
+                                                                    HasUpdateCommandLinePreviewEventHandlers {
+
+    interface AppTemplatePropertyEditorUiBinder extends UiBinder<Widget, AppTemplatePropertyEditor> {
+    }
 
-    @UiField(provided = true) AppTemplateContentPanel cp;
-    @UiField TextArea description;
-    @UiField TextField name;
+    @UiField(provided = true)
+    AppTemplateContentPanel cp;
+    @UiField
+    TextArea description;
+    @UiField
+    TextField name;
 
-    @Path("name") HeaderEditor nameEditor;
+    @Path("name")
+    HeaderEditor nameEditor;
     @Ignore
-    @UiField TextButton searchBtn;
+    @UiField
+    TextButton searchBtn;
     @Ignore
-    @UiField(provided = true) ToolSearchField tool;
-    @UiField FieldLabel toolLabel, appNameLabel, appDescriptionLabel;
-    @UiField(provided = true) PropertyEditorAppearance appearance;
-    private static AppTemplatePropertyEditorUiBinder BINDER = GWT.create(AppTemplatePropertyEditorUiBinder.class);
+    @UiField(provided = true)
+    ToolSearchField tool;
+    @UiField
+    FieldLabel toolLabel, appNameLabel, appDescriptionLabel;
+    @UiField(provided = true)
+    PropertyEditorAppearance appearance;
+
+    private static AppTemplatePropertyEditorUiBinder BINDER =
+            GWT.create(AppTemplatePropertyEditorUiBinder.class);
 
     private boolean labelOnlyEditMode = false;
 
     private AppTemplate model;
     Logger LOG = Logger.getLogger("App template");
 
-    @Inject AsyncProviderWrapper<DCListingDialog> dcListingDialogProvider;
+    @Inject
+    AsyncProviderWrapper<ToolListingDialog> dcListingDialogProvider;
+    @Inject
+    EventBus eventBus;
```

Comment:
I see it used in line `204` unless we are looking into different files / commits. I am going to change it name though to some thing like `toolListingDialogProvider`